### DISCUSSION TITLE
Can I remove DESIGN.md after add .design folder and readme.md inside the folder?

### DISCUSSION THREAD

**Discussion by cosmic-flood** [contribute-design/contribute.design#48] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/68597908/205257200-b93d108e-747e-49bf-bbc6-8f2048a99d6c.png)

**Comment by fivenp** [comment, marked as answer]
Yes... Well said @cosmic-flood! 
You can safely remove the `DESIGN.md` file once you created a `.design` folder with a `README.md` inside. 

* Having only a `DESIGN.md` is intended for rather small projects where all the guidelines can fit into one single file
* Having the `.design/` folder structure allows you to scale your documentation going forward

https://github.com/contribute-design/examples might give you an overview of how these guidelines can evolve over time (based on project type and how sophisticated the guides are :))
